packages/client/src/Tooltip.tsx

```diff
@@ -0,0 +1,16 @@
+import { Tooltip as TTip } from "@material-ui/core"
+import React, { ReactElement } from "react"
+
+export default function Tooltip({
```

Comment:
Nifty!

There is a more generalized way to wrap a component and pass through all of the props that it accepts. I looked at the type definition for `Tooltip` and saw that its props type is exported as `TooltipProps`. You want to accept any props that `Tooltip` can accept, but you want to pull out `enterDelay` and `leaveDelay` to set your own default values if the caller does not provide override values. The remaining props should be passed through. You can do that like this:

```ts
import { Tooltip as TTip } from "@material-ui/core"
import { TooltipProps } from "@material-ui/core/Tooltip"
import React from "react"

export default function Tooltip({
  enterDelay = 500,
  leaveDelay = 200,
  ...rest
}: TooltipProps) {
  return <TTip enterDelay={enterDelay} leaveDelay={leaveDelay} {...rest} />
}
```

Remember that `children` is just another prop, and it is passed through with everything else via `rest`.

Assigning default values when destructuring allows the caller to override `enterDelay` or `leaveDelay` if they want to.